packages/vscode/src/utils.ts

```diff
@@ -0,0 +1,62 @@
+import * as http from 'http';
+import { WorkspaceFolder } from 'vscode';
+import load from '@proload/core';
```

Comment:
This will match Astro's built-in support for `.ts` (etc) files! Don't think you need to add `@proload/plugin-typescript` since `@proload/plugin-tsm` handles this.

```suggestion
import load from '@proload/core';
import loadTypeScript from '@proload/plugin-tsm';

load.use([loadTypeScript]);
```

Reply:
So, I tried to use `load.use` and ran into a strange error about the function not existing. I marked this PR as WIP because I wanted to ask you about this 😅